src/App.js

```diff
@@ -16,7 +16,7 @@ function App() {
         <Route path="/" element={<Articles />} />
 
         <Route path="/articles/:article_id/comments" element={<Comments />} />
```

Comment:
Not too sure this needs to be an endpoint. Comments are displayed as part of the individual article, so having a separate endpoint doesn't feel necessary